core/tests/protractor/accessibility.js

```diff
@@ -110,6 +111,7 @@ describe('screenreader and keyboard user accessibility features', function() {
 
       // Should move the focus away from the elementToFocus.
       await browser.actions().sendKeys(protractor.Key.TAB).perform();
+      await browser.actions().sendKeys(protractor.Key.TAB).perform();
```

Comment:
Could you explain in your comment why you need to tab twice here? Right now it's a little confusing because it seems like a single tab should be enough to shift the focus away. Ditto below